app/main.py

```diff
@@ -1,6 +1,31 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+CITY = "Paris"
+URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = os.getenv("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    try:
+        response = requests.get(URL + f"?key={API_KEY}&q={CITY}")
+        response.raise_for_status()
+        data = response.json()
+
+        country = data.get("location").get("country")
+        localtime = data.get("location").get("localtime")
+        celsius = data.get("current").get("temp_c")
+        condition = data.get("current").get("condition").get("text")
```

Comment:
Consider adding checks to ensure that the keys 'location', 'current', and 'condition' exist in the response JSON before attempting to access them. This will prevent potential `AttributeError` if the API response structure changes or if there is an error in the response.